Makefile-docker

```diff
@@ -141,9 +141,9 @@ ui-tests:
 	python manage.py update_permissions_from_mc
 
 	# Create a proper superuser that can be used to access the API
-	python manage.py waffle_switch super-create-accounts on
-	python manage.py waffle_switch activate-autograph-signing on
-	python manage.py createsuperuser --email=uitest@mozilla.com --username=uitest --noinput --add-to-supercreate-group --save-api-credentials=tests/ui/variables.json --hostname=olympia.dev
+	python manage.py waffle_switch super-create-accounts on --create
```

Comment:
Did you move these waffle changes to another pull?  (Or did we just talk about it)

Reply:
I think we only talked about it, I'll back them out and commit them in a separate PR